### PR TITLE
Generator: Update SDK /services/postgresflex

### PR DESCRIPTION
**Pull request by stackit-pipeline**
Generated from GitHub run [9175239362](https://github.com/stackitcloud/stackit-api-specifications/actions/runs/9175239362)

### CODE REVIEW

File: services/postgresflex/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v0.13.1 (2024-05-22)
+
+- Remove unused model data types.
```
Comment (vicentepinto98) [inline services/postgresflex/CHANGELOG.md:3]:
```suggestion
- **Breaking change**: Remove unused model data types.
```

File: services/postgresflex/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## v0.13.1 (2024-05-22)
```
Comment (vicentepinto98) [inline services/postgresflex/CHANGELOG.md:1]: minor version upgrade since it has a breaking change

**Review by vicentepinto98 — COMMENTED** [review]
also add to the root CHANGELOG

File: services/postgresflex/CHANGELOG.md (hunk 2 above)
Comment (GokceGK) [reply services/postgresflex/CHANGELOG.md]: changed version to v0.14.0

**Review by vicentepinto98 — APPROVED** [review]

**Pull request merged**